template/python3/index.py

```diff
@@ -3,8 +3,15 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import sys
+import os
 from function import handler
 
+
+# distutils.util.strtobool() can throw an exception
+def is_true(val):
+    return val and val.lower() == "true" or val == "1"
+
+
 def get_stdin():
```

Comment:
I would guess that `get_stdin()` will produce an invalid set of bytes since it is reading lines, I would imagine it should attempt to read an entire payload as bytes. There may not even be any newlines in the binary input - i.e. a JPEG image.

Reply:
We need to see a test that verifies the input received in the function is a perfect match for the bytes inputted.

Reply:
If you accept the data in from a HTTP request (`curl --data-binary @input.jpg`), then write it to the container in /tmp you can use `kubectl cp -n openfaas-fn pod/name ./local-file.jpg` to get the file back and compare for instance.

Reply:
I have updated this PR and tested with image data and it is working fine. Image file written in container is rendering correctly.